src/API/src/db/shared/reference.resolver.ts

```diff
@@ -81,13 +93,25 @@ export class ReferenceResolver {
 
   @Query(() => PaginatedReferenceData)
   async allReferenceData(
-    @Args() { take, skip, orderBy, order }: GetReferenceDataArgs,
+    @Args()
+    {
+      take,
+      skip,
+      orderBy,
+      order,
+      startId,
+      endId,
+      textFilter,
+    }: GetReferenceDataArgs,
   ) {
     const { items, total } = await this.referenceService.findReferences(
       take,
       skip,
       orderBy,
       order,
+      startId,
+      endId,
+      textFilter,
```

Comment:
For newRef on line 130, should we have input.citation mapping to both citation and article_title?

Reply:
Citation stuff has to be thought about a bit more - I'll add a story to work it out.